spec/javascripts/test_spec.js

```diff
@@ -51,6 +51,16 @@ describe("Basic testing", function() {
     expect(blurredLocation.getPrecision()).toBe(1);
   });
 
+  it("Checks if getFullLat returns the full latitude of the map", function() {
+    blurredLocation.goTo(45.324324234,-53.32423234234,13);
+    expect(blurredLocation.getFullLat()).toBe(45.324324234);
+  });
+
+  it("Checks if getFullLon returns the full latitude of the map", function() {
+    blurredLocation.goTo(45.324324234,-53.32423234234,13);
```

Comment:
@jywarren Yeah though so, have set it according to zoom only. This one's ready to merge, is it?